Universities.txt

```diff
@@ -1,14 +1,14 @@
-Carleton University, 84.3, 85.0, 11755, 379, 45.3876, -75.6960
-University of Guelph, 87.3, 85.0,12806, 169, 43.5327, -80.2262
-Lakehead University, 81.9, 70.0, 8956, 337, 48.4211, -89.2607
-Laurentian University, 82.5, 75.0, 9461, 165, 46.4700, -80.9745
-McMaster University, 91.1, 85.0, 11194, 232, 43.2609, -79.9192
-University of Ottawa, 85.3, 75.0, 6332, 13, 45.4231, -75.6831
-Ontario Tech University, 81.3, 75.0, 12010, 126, 43.9458, -78.8960 
-Queen's University, 91.1, 85.0, 13243, 283, 44.2253, -76.4951
-Ryerson University, 86.6, 80.0, 11163, 743, 43.6577, -79.3788
-Toronto » All Campuses, 93.3, 85.0, 15864, 637,  43.6629, -79.3857, 
-University of Waterloo, 93.1, 85.0, 15115, 741, 43.4723, -80.5449, 
-Western » Main Campus, 89.6, 85.0, 13762, 332, 43.00995, -81.273613, 
-University of Windsor, 84.6, 75.0, 10638, 163, 42.3043, -83.0660
-York University, 84.3, 75.0, 14450, 772, 43.7735, -79.5019, 
\ No newline at end of file
+Carleton University, 84.3, 85.0, 11755, 379, 45.3876, -75.6960,
+University of Guelph, 87.3, 85.0,12806, 169, 43.5327, -80.2262,
+Lakehead University, 81.9, 70.0, 8956, 337, 48.4211, -89.2607,
+Laurentian University, 82.5, 75.0, 9461, 165, 46.4700, -80.9745,
+McMaster University, 91.1, 85.0, 11194, 232, 43.2609, -79.9192,
+University of Ottawa, 85.3, 75.0, 6332, 13, 45.4231, -75.6831,
+Ontario Tech University, 81.3, 75.0, 12010, 126, 43.9458, -78.8960,
+Queen's University, 91.1, 85.0, 13243, 283, 44.2253, -76.4951,
+Ryerson University, 86.6, 80.0, 11163, 743, 43.6577, -79.3788,
+Toronto » All Campuses, 93.3, 85.0, 15864, 637,  43.6629, -79.3857,
```

Comment:
Probably want to rename the "Toronto >> All Campuses" to "University of Toronto" or something